mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,55 @@
+import {useState} from "react";
+import styles from "src/component/select/Select.module.scss";
+
+export interface OptionType {
+  id: string;
+  value: string;
+  text: string;
+}
+
+/**
+   * Implement a simple select component
+   */
+export interface SelectProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Default select value (string)
+   */
+  value: string;
+  /**
+   * Select's name (string)
+   */
+  name: string;
+  /**
+   * Options list
+   */
+  options: OptionType[];
+}
+
+export const Select: React.FC<SelectProps> = (props: SelectProps) => {
+  const [selected, setSelected] = useState(props.value);
+  const handleOnChange = (e: React.ChangeEvent<HTMLSelectElement>) => setSelected(e.target.value);
+  return (
```

Comment:
Add an empty line between 34 and 35 lines

Reply:
done